spec/system/front_page_spec.rb

```diff
@@ -9,7 +9,7 @@
 
   it "has a header with links to helpful info" do
     visit "/"
-    expect(page).to have_link "About", href: "/about"
-    expect(page).to have_link "How to Submit", href: "/how-to-submit"
+    expect(page).to have_link "About"
+    expect(page).to have_link "How to Submit"
```

Comment:
This removes the check for the href. Seems like we should fix it instead, otherwise we won't know if it breaks.